### ISSUE TITLE
Move caption track for prerecorded audio to best practice

### ISSUE THREAD

**Issue by paulbohman** [dequelabs/axe-core#231]
The aXe rule to require a caption track for prerecorded audio is not a WCAG requirement. Prerecorded audio should have "an alternative" (according to 1.2.1), but the alternative can be a transcript. In fact, captions for prerecorded audio (assuming no video) are not particularly useful. Technique G158 ("Providing an alternative for time-based media for audio-only content") talks about transcripts, and does not mention captions at all: https://www.w3.org/WAI/GL/2016/WD-WCAG20-TECHS-20160105/G158

Since this rule really ought to be looking for a transcript, and since there is no way to reliably tell if an audio file has a transcript, we probably need to drop this rule entirely.

**Comment by dylanb** [comment]
Captions are definitely better for certain types of users, should we demote this to a best-practice?

**Comment by paulbohman** [comment]
I'm curious who you're thinking of that would benefit from captions for audio-only content, or what use cases would benefit most from synchronous captions. For live events, I would say live captions are absolutely beneficial, because they're synchronized with an actual sequence of events, but with pre-recorded audio only, it's not as obvious to me who would benefit and under what circumstances.

**Comment by dylanb** [comment]
People who are hard of hearing but not completely deaf and who might need the captions to see what they did not quite pick up on. Captions are synched with the rest of the audio including music for example.

**Comment by paulbohman** [comment]
Ok. I agree that there could be some benefit for people who are hard of hearing who may not need the captions for every aspect of the audio experience, but who might need it to confirm what they're hearing. It's pretty rare to see captions on pre-recorded audio though. I looked at the "Understanding 1.2.2" document, and all the examples they give there are for multimedia (video + audio) https://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-captions.html 

You've convinced me that the use case exists, and that there is a benefit, but the assumption -- by me and the W3C documentation -- is that transcripts cover this use case pretty well too, even if not perfectly under all circumstances. So I'd move it off of the requirement list and onto the best practice list, and I'd specify that the impact is on the low side, assuming there is a transcript.

**Comment by marcysutton** [comment]
We've improved the rule for this since it introduced false positives. That work has been merged, stay tuned for axe-core 2.2.0.
